src/Steppers/Steppers.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2015 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import React, { FC } from 'react';
+
+import classNames from '../utils/classNames';
+
+import './steppers.scss';
+
+export type Step = {
+    title: string;
+    caption?: string;
+    active?: boolean;
+    success?: boolean;
+    warn?: boolean;
+    fail?: boolean;
+};
+
+export interface Props {
+    title?: string;
+    steps: Step[];
+}
+
+const Steppers: FC<Props> = ({ title, steps }) => (
+    <>
+        {title && <div>{title}</div>}
```

Comment:
There are different preferences on this topic, but I prefer
```suggestion
        {title != null ? <div>{title}</div> : null}
```
And we could also check if the string is empty, because then it would not make sense to render the title, right?
```suggestion
        { (title != null && title !== '') ? <div>{title}</div> : null}
```